frontend/src/pages/Interventions/data/reducer.ts

```diff
@@ -1,13 +1,14 @@
-import {Action, ActionTypes, InterventionState} from "../../../models/types";
-import {RootStateOrAny} from "react-redux";
+import { Action, ActionTypes, InterventionState } from "../../../models/types";
+import { RootStateOrAny } from "react-redux";
 
 const initialState: InterventionState = {
-  currentWordIdx: 0,
-  currentActivityIdx: 0,
   interventions: null,
```

Comment:
ah so right now it is like interventions.interventions which is a bit repetitive. Now the reducer looks like 
`
interventions: {
  interventions: {
    wordIdx .. .. ..
  }
}
`
Is it ok to destruct it to be something like what we have in notion? So like 
`
interventions: {
		activityIdx: 0,
		wordIdx: 0,
		wordList: [
			{
				activities: {
						a1: x, a2: y, a3: z, a4: w 
					} 
			},
			{ ... },
		]
}
`